src/sif/truckcost.cc

```diff
@@ -268,12 +270,13 @@ class TruckCost : public DynamicCost {
   float low_class_penalty_;  // Penalty (seconds) to go to residential or service road
 
   // Vehicle attributes (used for special restrictions and costing)
-  bool hazmat_;     // Carrying hazardous materials
-  float weight_;    // Vehicle weight in metric tons
-  float axle_load_; // Axle load weight in metric tons
-  float height_;    // Vehicle height in meters
-  float width_;     // Vehicle width in meters
-  float length_;    // Vehicle length in meters
+  bool hazmat_;         // Carrying hazardous materials
+  float weight_;        // Vehicle weight in metric tons
+  float axle_load_;     // Axle load weight in metric tons
+  float height_;        // Vehicle height in meters
+  float width_;         // Vehicle width in meters
+  float length_;        // Vehicle length in meters
+  uint32_t axle_count_; // Vehicle axle count
```

Comment:
can use `uint8_t` here and for the range above

Reply:
changed it to uint8_t